libs/tree-sitter-wing/grammar.js

```diff
@@ -349,9 +350,19 @@ module.exports = grammar({
         $.struct_literal,
         $.optional_test,
         $.compiler_dbg_panic,
-        $.optional_unwrap
+        $.optional_unwrap,
+        $.intrinsic
       ),
 
+    intrinsic: ($) =>
+      prec.right(
+        seq(
+          field("name", $.intrinsic_identifier),
+          optional(field("args", $.argument_list))
+        )
+      ),
+    intrinsic_identifier: ($) => /@[a-zA-Z]*/,
```

Comment:
No underscores or numbers? Should probably be same as any old identifier. Not critical I guess.